JSDemos/Demos/PivotGrid/OLAPDataSource/description.md

```diff
@@ -1 +1,18 @@
-The PivotGrid supports OLAP services (MS&nbsp;SQL Server Analysis Services). This allows you to&nbsp;deliver solutions that can easily analyze millions of&nbsp;data rows within your app. In&nbsp;this demo, the PivotGrid uses **XmlaStore** to&nbsp;connect the component to&nbsp;a&nbsp;sample remote OLAP service. The Field Chooser is&nbsp;configured to&nbsp;show the search box to&nbsp;speed up&nbsp;choosing the required field.
\ No newline at end of file
+The [PivotGrid](/Documentation/Guide/UI_Components/PivotGrid) component supports OLAP services (MS SQL Server Analysis Services). This demo shows how to use a remote OLAP cube as the PivotGrid's data source.
```

Comment:
- No reason for the PivotGrid link: the reader already sees the component in the demo.
- MS stands for Microsoft. We were told not to abbreviate this.

```suggestion
The PivotGrid component supports OLAP services (Microsoft SQL Server Analysis Services). This demo shows how to use a remote OLAP cube as the PivotGrid's data source.
```